.github/workflows/acsf-console-actions.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - name: Install composer dependencies
         run: |
           composer require --dev php-coveralls/php-coveralls
+          composer require --dev drupal/coder
```

Comment:
Could you please add this to composer.json as well to the dev deps? And by that, could you please confirm that without requiring the package here, a simple `composer install` will not do the work, I.e. do you still need this line even after adding the package to dev deps?

Reply:
We already have it in helpers' composer.json in require-dev section, do you think we need it in all the libraries separately?

Reply:
Yes, if I'm not mistaken, composer will not install the dev deps of dependencies by default.

Reply:
And if you include drupal/coder in composer.json this won't be needed.

Reply:
Yes you are right, dev dependencies need to be present for each package separately. I'll test the scenario with composer.json.

Reply:
@saxumVermes drupal/coder gets installed if added to require-dev in composer.json but it starts to throw a different error on Github Actions:
`Run ./vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer
Using config file: /home/runner/work/acsf-contenthub-console/acsf-contenthub-console/vendor/squizlabs/php_codesniffer/CodeSniffer.conf

Config value "installed_paths" updated successfully; old value was "../../drupal/coder/coder_sniffer,../../sirbrillig/phpcs-variable-analysis,../../slevomat/coding-standard"
ERROR: Referenced sniff "SlevomatCodingStandard.ControlStructures.RequireNullCoalesceOperator" does not exist

Run "phpcs --help" for usage information

Error: Process completed with exit code 3.`

Same error appears for helpers library as well. If added to workflow yml it works fine. Do you think we can go that way where we remove this from composer.json from helpers library as well and just add it dynamically on GA.

Reply:
@amangrover90  I think it should be good now.